app/views/announcements/index.html.slim

```diff
@@ -0,0 +1,5 @@
+div.page-header
+  h1
+    = t('.header')
+= render partial: 'admin/announcements/announcement', collection: @announcements
```

Comment:
This doesn't seem to work: `Missing partial system_announcements/_system_announcement`. Is it looking for the partial based on the types of the items in the collection?